src/Nevermind/Nevermind.Json/IJsonSerializer.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Nevermind.Json
+{
+    public interface IJsonSerializer
+    {
+        T DeserializeAnonymousType<T>(string json, T definition);
```

Comment:
let us review if the test projects should use this as well, might be that a similar interface is already in test namespaces